aci/resource_aci_fvbd.go

```diff
@@ -313,7 +313,7 @@ func getRemoteBridgeDomain(client *client.Client, dn string) (*models.BridgeDoma
 	fvBD := models.BridgeDomainFromContainer(fvBDCont)
 
 	if fvBD.DistinguishedName == "" {
-		return nil, fmt.Errorf("BridgeDomain %s not found", fvBD.DistinguishedName)
+		return nil, fmt.Errorf("Bridge Domain %s not found", dn)
```

Comment:
I saw earlier mentioning of BD, should we use abbreviated forms or full form? This is generic question that applies for all other object mentions. @lhercot any thoughts?

Reply:
Leaving it as Bridge Domain, seeing that the name of the function is the same